site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -445,6 +448,64 @@ For high availability deployments with purchased support:
 * The FusionAuth engineering team will consult on product implementation and can offer "best practices" advice to further ensure system stability.
 * We size the instance properly for expected traffic, in consultation with the customer.
 
+== Deployment IP Addresses
+
+You may need to know the IP addresses of the instances in your FusionAuth Cloud deployment. There are any number of reasons you might need this information, including:
+
+* Adding the IP addresses to the allow list of an email deliverability vendor.
+* Punching a hole in a corporate firewall to allow for webhook delivery.
+* Adding the IP address to a network access control list to enable access to an on-premises LDAP directory with a connector.
+
+[WARNING.warning]
+====
+The IP addresses of FusionAuth Cloud instances may change at any time. Prepare for this by scripting this procedure, running it regularly, and updating the allow list.
+
+Hostnames are immutable, and if you can use them instead of IP addresses, that will require less maintenance.
```

Comment:
> Hostnames are immutable

Is it hostnames. Or is it 

> the DNS of the FusionAuth instances are immutable.

Or am I splitting hairs\saying the same thing 😄

Reply:
I'd say it is hostnames.